backend/dataall/modules/notebooks/db/models.py

```diff
@@ -1,11 +1,14 @@
+"""ORM models for sagemaker notebooks"""
+
 from sqlalchemy import Column, String, Integer
-from sqlalchemy.orm import query_expression
 
-from .. import Base
-from .. import Resource, utils
+from dataall.db import Base
+from dataall.db import Resource, utils
 
 
 class SagemakerNotebook(Resource, Base):
```

Comment:
Shouldn't db layer be part of each module? So if I want to introduce a new module for instance a SagemakerNotebook related one I would have to define a db model/migration script for it in a separate directory for a particular module to not mix all the stuff together in one db/models directory?